cmd/xl-v1-list-objects-heal.go

```diff
@@ -325,15 +325,14 @@ func (xl xlObjects) listMultipartUploadsHeal(bucket, prefix, keyMarker,
 		}
 		// Collect uploads until maxUploads limit is reached.
 		for walkResult := range walkerCh {
-			// Ignore errors like errDiskNotFound
-			// and errFileNotFound.
-			if isErrIgnored(walkResult.err,
-				xlTreeWalkIgnoredErrs...) {
-				continue
-			}
-			// For any error during tree walk we should
-			// return right away.
+
+			// For any error during tree walk, except for file not found,
```

Comment:
> Can you explain this with a test case ?

Done, I've added some tests in admin-handlers_test.go: Test 1 and 2 succeed in this PR but not in the current master. isTruncated is supposed to return False but it is returning True.